src/templates/edit.tsx

```diff
@@ -57,6 +57,20 @@ const Edit: () => JSX.Element = () => {
   const [entityDocument, setEntityDocument] = useState<any>(); // json data
   const [saveState, setSaveState] = useState<SaveState>();
   const [saveStateFetched, setSaveStateFetched] = useState<boolean>(false); // needed because saveState can be empty
+  const [hasParent, setHasParent] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (window.parent.name) {
+      setHasParent(true);
+    }
+  }, [setHasParent]);
```

Comment:
I don't think you want the setter. Just empty array. Double check that it works though with the change.

Reply:
Done